gum/gumexceptor.c

```diff
@@ -29,13 +32,18 @@
 # include <signal.h>
 #endif
 
+#ifdef HAVE_QNX
+# include <sys/debug.h>
+# include <unix.h>
+#endif
+
 typedef struct _GumExceptionHandlerEntry GumExceptionHandlerEntry;
 #if defined (G_OS_WIN32) || defined (HAVE_DARWIN)
 # define GUM_NATIVE_SETJMP setjmp
 # define GUM_NATIVE_LONGJMP longjmp
   typedef jmp_buf GumExceptorNativeJmpBuf;
 #else
-# ifdef sigsetjmp
+# ifdef __sigsetjmp
```

Comment:
any suggestions for how to implement this? There doesn't appear to be a __sigsetjmp on qnx